impl/src/main/java/org/ehcache/impl/config/store/disk/OffHeapDiskStoreConfiguration.java

```diff
@@ -61,6 +80,15 @@ public int getWriterConcurrency() {
   }
 
   /**
+   * Returns the number of disk segments created
+   *
+   * @return the number of segments
+   */
+  public int getDiskConcurrency() {
```

Comment:
I would call this `getDiskSegments` or `getDiskSegmenting` since that's the name used in javadoc and it would prevent confusion with the `writerConcurrency`.